api/pom.xml

```diff
@@ -13,7 +13,7 @@
    <parent>
       <groupId>org.openmrs</groupId>
       <artifactId>openmrs</artifactId>
-      <version>2.1.0-SNAPSHOT</version>
+      <version>2.1.4-SNAPSHOT</version>
```

Comment:
This shouldn't also be going in master

Reply:
Ah! Apologies, this should definitely not go into master. I branched out from 2.x and incorrectly created PR for master. Gonna correct this. Thanks for pointing out.